cid-redirects.json

```diff
@@ -2977,6 +2977,7 @@
   "/Visualizations-and-Alerts/Dashboard_(New)/Panels/Area_Charts": "/docs/dashboards-new/panels/area-charts",
   "/Visualizations-and-Alerts/Dashboard_(New)/Panels/Bar_Charts": "/docs/dashboards-new/panels/bar-charts",
   "/Visualizations-and-Alerts/Dashboard_(New)/Panels/Box_Plot_Charts": "/docs/dashboards-new/panels/box-plot-charts",
+  "/Visualizations-and-Alerts/Dashboard_(New)/Panels/Sankey_Charts": "/docs/dashboards-new/panels/sankey-charts",
```

Comment:
I don't believe this page existed in the legacy site, so there'd be nothing to redirect from. Just checked the last build (Oct 2022) and don't see it there. Can you confirm?

Reply:
@JV0812 are these product redirects from Chung? If so, they should really use the existing URL.

Reply:
There was no existing redirect link and also the file was not existing in the legacy site, I added this as it was a new doc.

Reply:
Thanks for clarifying, @JV0812.

Since the old URL (/Visualizations-and-Alerts/Dashboard_(New)/Panels/Sankey_Charts) never existed in the first place, you delete this line. This file is only used to redirect old URLs to new URLs. Just need to add to `sidebars.ts`.

Reply:
@kimsauce Thanks Kim. The changes are deleted.

Reply:
@JV0812 thank you! once you click **Resolve conversation** below, we can merge this